SuperBuild/External_CCPi-Framework.cmake

```
@@ -0,0 +1,135 @@
+  if("${PYTHON_STRATEGY}" STREQUAL "PYTHONPATH")
+    # in case of PYTHONPATH it is sufficient to copy the files to the 
+    # $PYTHONPATH directory
+  ExternalProject_Add(${proj}
+    ${${proj}_EP_ARGS}
+    GIT_REPOSITORY ${${proj}_URL}
+    GIT_TAG ${${proj}_TAG}
+    SOURCE_DIR ${${proj}_SOURCE_DIR}
+    BINARY_DIR ${${proj}_BINARY_DIR}
+    DOWNLOAD_DIR ${${proj}_DOWNLOAD_DIR}
+    STAMP_DIR ${${proj}_STAMP_DIR}
+    TMP_DIR ${${proj}_TMP_DIR}
+    INSTALL_DIR ${libcilreg_Install_Dir}
+    
+    CONFIGURE_COMMAND ""
+    BUILD_COMMAND ""
+    INSTALL_COMMAND ${CMAKE_COMMAND} -E copy_directory ${${proj}_SOURCE_DIR}/Wrappers/Python/ccpi ${PYTHON_DEST}/ccpi
+    #TEST_COMMAND ${PYTHON_EXECUTABLE} -m unittest discover -s ${${proj}_SOURCE_DIR}/Wrappers/Python/test/ -p test_Data*.py &&
+    #${PYTHON_EXECUTABLE} -m unittest discover -s ${${proj}_SOURCE_DIR}/Wrappers/Python/test/ -p test_algor*.py &&
+    #${PYTHON_EXECUTABLE} -m unittest discover -s ${${proj}_SOURCE_DIR}/Wrappers/Python/test/ -p test_run_*.py
+    CMAKE_ARGS
+        -DCMAKE_INSTALL_PREFIX=${libcilreg_Install_Dir}
+    DEPENDS
+        ${${proj}_DEPENDENCIES}
+  )
+
+  else()
+    # if SETUP_PY one can launch the conda build.sh script setting 
+    # the appropriate variables.
+  ExternalProject_Add(${proj}
+    ${${proj}_EP_ARGS}
+    GIT_REPOSITORY ${${proj}_URL}
+    GIT_TAG ${${proj}_TAG}
+    SOURCE_DIR ${${proj}_SOURCE_DIR}
+    BINARY_DIR ${${proj}_BINARY_DIR}
+    DOWNLOAD_DIR ${${proj}_DOWNLOAD_DIR}
+    STAMP_DIR ${${proj}_STAMP_DIR}
+    TMP_DIR ${${proj}_TMP_DIR}
+    INSTALL_DIR ${libcilreg_Install_Dir}
+    
+    CONFIGURE_COMMAND ""
+    BUILD_COMMAND ""
+    INSTALL_COMMAND ${CMAKE_COMMAND} -E env CIL_VERSION=${CIL_VERSION} SRC_DIR=${${proj}_BINARY_DIR} RECIPE_DIR=${${proj}_SOURCE_DIR}/Wrappers/Python/conda-recipe PYTHON=${PYTHON_EXECUTABLE} bash ${${proj}_SOURCE_DIR}/Wrappers/Python/conda-recipe/build.sh
+    CMAKE_ARGS
+        -DCMAKE_INSTALL_PREFIX=${libcilreg_Install_Dir}
+    DEPENDS
+        ${${proj}_DEPENDENCIES}
+  )
+  endif()
+
+
+    set(${proj}_ROOT        ${${proj}_SOURCE_DIR})
+    set(${proj}_INCLUDE_DIR ${${proj}_SOURCE_DIR})
+
+    add_test(NAME CIL_FRAMEWORK_TESTS_1
+             COMMAND ${PYTHON_EXECUTABLE} -m unittest discover -p test_Data*.py 
+    WORKING_DIRECTORY ${${proj}_SOURCE_DIR}/Wrappers/Python/test)
+    add_test(NAME CIL_FRAMEWORK_TESTS_2
+             COMMAND ${PYTHON_EXECUTABLE} -m unittest discover -p test_algor*.p 
+    WORKING_DIRECTORY ${${proj}_SOURCE_DIR}/Wrappers/Python/test)
+    add_test(NAME CIL_FRAMEWORK_TESTS_3
+             COMMAND ${PYTHON_EXECUTABLE} -m unittest discover -p test_run_*.py
+    WORKING_DIRECTORY ${${proj}_SOURCE_DIR}/Wrappers/Python/test)
+     
+   else()
+      if(${USE_SYSTEM_${externalProjName}})
+        find_package(${proj} ${${externalProjName}_REQUIRED_VERSION} REQUIRED)
+        message("USING the system ${externalProjName}, set ${externalProjName}_DIR=${${externalProjName}_DIR}")
```

```suggestion
        message("USING the system ${externalProjName}")
```
I don't think the `_DIR` variable is used at all (no CMake config)